src/github3/pulls.py

```diff
@@ -575,6 +576,15 @@ def reviews(self, number=-1, etag=None):
         url = self._build_url('reviews', base_url=self._api)
         return self._iter(int(number), url, PullReview, etag=etag)
 
+    def combined_status(self):
+        """ Retrieve the combined status for this Pull Request
+        :rtype:
+            :class:`~github3.repos.CombinedStatus`
```

Comment:
This would need a `.. versionadded:: 1.3.0` to reflect when this function was added to the API.